feat: add vite bundler documentation to admin panel customisation (#1963)
strapi/documentation: docusaurus/docs/dev-docs/admin-panel-customization.md

const captionStyle = {fontSize: '12px'}
const imgStyle = {width: '100%', margin: '0' }

> The admin panel is a React based SPA (single-page-application). It encapsulates all the installed plugins of a Strapi application. Some of its aspects can be [customized](#customization-options), and plugins can also [extend](#extension) it.

Reviewer: ```suggestion
The admin panel is a React-based single-page application. It encapsulates all the installed plugins of a Strapi application. Some of its aspects can be [customized](#customization-options), and plugins can also [extend](#extension) it.
```

- rename the default `app.example.js` file into `app.js`,
- and create a new `extensions` folder in `./src/admin/`. Strapi projects already contain by default another `extensions` folder in `./src/` but it is for plugins extensions only (see [Plugins extension](/dev-docs/plugins-extension)).

>   :::

Reviewer: ```suggestion
:::
```

Email templates should be edited through the admin panel, using the [Users and Permissions plugin settings](/user-docs/settings/configuring-users-permissions-plugin-settings#configuring-email-templates).

## Bundlers (experimental)

> We offer two different bundlers to be used with your Strapi app, [webpack](#webpack) and [vite](#vite).

Reviewer: ```suggestion
2 different bundlers can be used with your Strapi application, [webpack](#webpack) and [vite](#vite).
```

};
```

</TabItem>

<TabItem value="ts" label="TypeScript">

```ts title="./my-app/src/admin/webpack.config.ts"
export default (config, webpack) => {
  // Note: we provide webpack above so you should not `require` it

  // Perform customizations to webpack config
  config.plugins.push(new webpack.IgnorePlugin(/\/__tests__\//));

  // Important: return the modified config
  return config;
};
```

</TabItem>
</Tabs>

### Vite

> :::warning

Reviewer: ```suggestion
:::caution
```
